lib/storage/InstanceSync/index.ts

```diff
@@ -0,0 +1,16 @@
+import NOOP from 'lodash/noop';
+
+/**
+ *  This is used to keep multiple browser tabs in sync, therefore only needed on web
+ *  On native platforms, we omit this syncing logic by setting this to mock implementation.
+ */
+const InstanceSync = {
+    init: NOOP,
```

Comment:
Does this need to use lodash, or can it just be an empty arrow function like `init: () => {}`?

Reply:
@tgolen if I add empty functions I'm getting eslint errors:

<img width="586" alt="image" src="https://github.com/Expensify/react-native-onyx/assets/22820318/edba71a3-4169-4fa1-b241-b2961d781d44">

Do you think it would be better to specify our own function?

Reply:
Hm, that's interesting. OK, it's probably fine to leave it as `NOOP` for now. Thanks for trying!